.gitignore

```diff
@@ -1,3 +1,4 @@
+data/config.js
 src/config.js
```

Comment:
should we remove src/config.js?

Reply:
```
src/config.js
src/node_modules/
src/package-lock.json
```

Are all old locations of files, should probably all be removed as long as the duplicate files have been deleted locally already.

Reply:
Let's assume people know what they are doing :^)